packages/SwingSet/src/kernel/liveSlots.js

```diff
@@ -849,6 +835,10 @@ function build(
       if (o) {
         exportedRemotables.delete(o);
       }
+      const { virtual } = parseVatSlot(vref);
```

Comment:
not for today, but a future refactoring to make: since this is now the third time `vref` is getting parsed, we should probably rearrange this to have a top-level loop on `vrefs`, parse it once inside the loop, then have all three checks (type, allocatedByVat, virtual/not-virtual) after that parse